src/features/identityCheck/api/useGetStepperInfo.ts

```diff
@@ -12,6 +13,7 @@ export const useGetStepperInfo = (): {
   identificationMethods?: SubscriptionStepperResponse['allowedIdentityCheckMethods'] | null
 } => {
   const { data } = useQuery([QueryKeys.STEPPER_INFO], () => api.getNativeV1SubscriptionStepper())
+  eventMonitoring.setExtras({ useGetStepperInfoResponse: JSON.stringify(data) })
```

Comment:
ça pourrait etre bien d'ajouter un todo

```suggestion
  eventMonitoring.setExtras({ useGetStepperInfoResponse: JSON.stringify(data) }) # TODO(PC-XXX): supprimer cette ligne une fois que le bug est corrigé
```

Reply:
Ahah le gars commente avec `#` en JS, bizarre le Tech Lead du Pass 😛

Reply:
Tech Lead != compétent 😜